services/skus/controllers.go

```diff
@@ -3,33 +3,42 @@ package skus
 import (
```

Comment:
I'm not sure what's going on with these or how this refactor is going to affect the nitro-payments branch